index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    
+    this.items = [];
+    this.length = this.items.length;
+    this.itemsOrdered = this.items.sort();
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort((a, b) => a - b)
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.length || pos < 0) {
+      throw new Error("OutOfBounds");
+    } else {
+      return this.items[pos]
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error("Empty SortedList");
+    } else { 
+      const max = this.itemsOrdered.length - 1;
+      return this.itemsOrdered[max];
```

Comment:
Ingeniosa forma de hacerlooo 🚀 
Tambien aqui lo que podemos hacer es utilizar la función max() retorna el valor máximo presente en la lista. Para hacer esto 🔥 , utiliza el método Math.max(...this.items), que toma una cantidad variable de argumentos y devuelve el valor más grande entre ellos. En este caso, el operador de propagación ... se utiliza para pasar los elementos individuales de la lista como argumentos a la función Math.max().
```
 return Math.max(...this.items)
 ```